application/dash_application/views/SARS-CoV-2.py

```diff
@@ -123,9 +124,13 @@
 # Build lists of attributes for sorting, shaping, and filtering on
 ALL_PROJECTS = util.unique_set(BEDTOOLS_DF, PINERY_COL.StudyTitle)
 ALL_KITS = util.unique_set(BEDTOOLS_DF, PINERY_COL.PrepKit)
-# TODO: Remove kludge and uncomment real line after Pinery is merged
-ILLUMINA_INSTRUMENT_MODELS = util.unique_set(BEDTOOLS_DF, PINERY_COL.InstrumentName)
-#ILLUMINA_INSTRUMENT_MODELS = list(util.get_illumina_instruments(RNA_DF))
+
+BEDTOOLS_CALC_DF = util.df_with_instrument_model(BEDTOOLS_CALC_DF, PINERY_COL.SequencerRunName)
```

Comment:
`BEDTOOLS_CALC_DF` is not used in the view. DataFrames, with the exception of coverage percentile, are merged into `BEDTOOLS_DF`. This should be merged with the instrument data.